services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/MongoDbDeviceRegistryMetrics.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.deviceregistry.mongodb;
+
+import org.eclipse.hono.service.DeviceRegistryMetrics;
+import org.eclipse.hono.service.metric.NoopBasedMetrics;
+
+/**
+ * Metrics for the MongoDB based Device Registry service.
+ */
+public interface MongoDbDeviceRegistryMetrics extends DeviceRegistryMetrics {
```

Comment:
this interface doesn't seem to serve any purpose in addition to *DeviceRegistryMetrics*, does it?